pytest_reportportal/service.py

```diff
@@ -240,6 +240,31 @@ def start_pytest_item(self, test_item=None):
         log.debug('ReportPortal - Start TestItem: request_body=%s', start_rq)
         self.RP.start_test_item(**start_rq)
 
+    def is_client_support_item_update(self):
```

Comment:
```suggestion
    def is_item_update_supported(self):
```
Let's keep our text close to the language rules.